test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,70 @@
+require('dotenv').config();
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'Ktagudelo';
+
+const searchGist = async () => {
+  const response = await axios.get(`${urlBase}/users/${githubUserName}/gists`, {
+    headers: {
+      Authorization: `token ${process.env.ACCESS_TOKEN}`
+    }
+  });
+
+  const search = response.data.find(
+    (element) => element.description === 'Example of a gist'
+  );
+  return search.id;
+};
+
+describe('GitHub User Create And Delete Gist', () => {
+  it('Consume POST GitHub User Create Gist', async () => {
+    const responseGist = await axios.post(
+      `${urlBase}/gists`,
+      {
+        description: 'Example of a gist',
+        public: false,
+        files: {
+          'README.md': {
+            content: 'First Gist'
+          }
+        }
+      },
+      {
+        headers: {
+          Authorization: `token ${process.env.ACCESS_TOKEN}`
+        }
+      }
+    );
+    expect(responseGist.status).to.containSubset(StatusCodes.CREATED);
+  });
+
+  it('Consume GET GitHub User List Gist', async () => {
+    const responseGetGist = await axios.get(
+      `${urlBase}/users/${githubUserName}/gists`,
+      {
+        headers: {
+          Authorization: `token ${process.env.ACCESS_TOKEN}`
+        }
+      }
+    );
+    expect(responseGetGist.status).to.containSubset(StatusCodes.OK);
+  });
```

Comment:
this call needs to be done after the delete is done, and you have to verify that the response status is an 404 error code.